terraform/cloud_init.cfg

```diff
@@ -0,0 +1,17 @@
+#cloud-config
+hostname: goodbit_host
+local-hostname: ubuntu-01
+instance-id: ubuntu01
+users:
+- default
+- name: goodbit
+  primary_group: goodbit
```

Comment:
check if this can be taken from variables

Reply:
https://grantorchard.com/dynamic-cloudinit-content-with-terraform-file-templates/